FRENCH/src/ch15-01-box.md

```diff
@@ -0,0 +1,661 @@
+themselves in this way very often. Having values like a single `i32` on the
+stack, where they’re stored by default, is more appropriate in the majority of
+situations. Let’s look at a case where boxes allow us to define types that we
+wouldn’t be allowed to if we didn’t have boxes.
+-->
+
+Déposer une seule valeur sur le tas n'est pas très utile, donc vous n'utiliserez
+très rarement les boites de cette manière. Laisser les valeurs comme des `i32`
+indépendantes sur la pile, où elles sont stockées par défaut, reste plus
+approprié dans la majeure partie des situations. Regardons un cas où les boites
+nous permettent de définir des types que nous pourrions par le faire si nous
+n'avions pas les boites.
+
+<!--
+### Enabling Recursive Types with Boxes
+-->
+
+### Pouvoir utiliser des types récursifs grâce aux boites
+
+<!--
+At compile time, Rust needs to know how much space a type takes up. One type
+whose size can’t be known at compile time is a *recursive type*, where a value
+can have as part of itself another value of the same type. Because this nesting
+of values could theoretically continue infinitely, Rust doesn’t know how much
+space a value of a recursive type needs. However, boxes have a known size, so
+by inserting a box in a recursive type definition, you can have recursive types.
+-->
+
+Au moment de la compilation, Rust a besoin de savoir combien s'espace prend un
+type. Un des types dont la taille ne peut pas être connu au moment de la
+compilation est le *type récursif*, dans lequel une valeur peut avoir une partie
+de sa définition qui a une valeur du même type qu'elle-même. Comme emboîtement
+de valeurs pourrait théoriquement se poursuivre à l'infini, Rust ne sait pas
+combien d'espace une valeur d'un type récursif peut avoir besoin. Cependant, les
+boites ont une taille connue, donc en utilisant une boite dans la définition
+d'un type récursif, vous pouvez créer des types récursifs.
+
+<!--
+Let’s explore the *cons list*, which is a data type common in functional
+programming languages, as an example of a recursive type. The cons list type
+we’ll define is straightforward except for the recursion; therefore, the
+concepts in the example we’ll work with will be useful any time you get into
+more complex situations involving recursive types.
+-->
+
+Découvrons maintenant la *liste de construction* (NdT : cons list), qui est un
+type de donnée courant dans les langages de programmation fonctionnels, comme
+étant un exemple de type récursif. Le type liste de construction que nous allons
+définir est plutôt simple, sauf pour les cas de récursivité ; par conséquent,
+les concepts dans l'exemple avec lequel nous allons travailler vous seront
+utiles à chaque fois que vous vous retrouverez dans des situations plus
+complexes qui impliquent des types récursifs.
+
+<!--
+#### More Information About the Cons List
+-->
+
+#### En savoir plus sur kes listes de construction
+
```

Comment:
Replace :
```suggestion
#### En savoir plus sur kes listes de construction

```
👎 ... sur ***kes*** listes 
by :
👍 ... sur ***les*** listes ...

Reply:
Thank you, @itotutona ! 😉
_(i've edited your suggestion to directly commit it from GitHub)_